src/VehicleInterface.c

```diff
@@ -399,6 +400,151 @@ void vsc_send_heartbeat(VscInterfaceType* vscInterface, uint8_t EStopStatus) {
 
 }
 
+/** vsc_scm_target_set()
+ * @brief This function sends a SCM Target Set Command to the VSC to prepare for a settings grab.
+ * 
+ * @param vscInterface VSC Interface Structure.
+ * @param target_id The identifier for the target, 0 for local.
+ */
+void vsc_scm_target_set(VscInterfaceType* vscInterface, int target_id)
+{
+	VscMsgType scmTargetSetMsg;
+	SCMTargetSetMsgType *msgPtr = (SCMTargetSetMsgType*) scmTargetSetMsg.msg.data;
+
+	/* Fill Message */
+	scmTargetSetMsg.msg.msgType = MSG_SCM_TARGET_SET;
+	scmTargetSetMsg.msg.length = sizeof(SCMTargetSetMsgType);
+
+	/* Fill in the Target ID */
+	msgPtr->target_id = target_id;
+
+	/* Send Message */
+	if (vsc_send_msg(vscInterface, &scmTargetSetMsg) < 0) {
+    // Error print commented out due to spam
+	  // fprintf(stderr, "vsc_example: Send Message Failure (Errno: %i)\n", errno);
+	}
```

Comment:
Is there something that you want to happen when this error occurs? Or will you print out some message in the future? Otherwise we can we remove the if statement and just send the message?

Reply:
That error handling was inherited from how the other previously existing handler functions handled the errors. I'm going to refer to the judgement of the previous developer here, so that we have an easy way of debugging if something doesn't work in the future.

Reply:
I would prefer if it actually printed.

Reply:
Did some testing with printing, and what I was afraid of came true... I tested some VSC disconnections, and the message spams at a 50Hz rate, clogging up the tail log. I think it might just be better to remove it altogether like @taylorwentzel suggested. It's not like we're going to get any real use our of it...